src/controllers/DeliveryEmployeeController.ts

```diff
@@ -0,0 +1,49 @@
+import express from "express";
+import { createDeliveryEmployee, deleteDeliveryEmployee, editDeliveryEmployee, getDeliveryEmployee, getDeliveryEmployees } from "../services/DeliveryEmployeeService";
+
+export const getDeliveryEmployeeList = async (req: express.Request, res: express.Response): Promise<void> => {
```

Comment:
```suggestion
export const getDeliveryEmployeeList = async (req: Request, res: Response): Promise<void> => {
```
same for all other functions please